README.md

```diff
@@ -1,24 +1,74 @@
-# README
+![default-monochrome](https://user-images.githubusercontent.com/56429354/205527587-ac4a2eee-a331-4493-ab17-744ae465e646.svg#gh-light-mode-only)
+![default](https://user-images.githubusercontent.com/56429354/205528723-bced613e-619a-4c0f-aa30-e4c53ce7e501.svg#gh-dark-mode-only)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+>Book car is a platform to book a car that is present in different cities for a given period of time. The user selects the duration of rental per car model type and price range. if a user is an admin he can manage the car resources by updating the list of cars in the database.
 
-Things you may want to cover:
+## Frontend Repository
+[GitHub Link](https://github.com/Gedewon/CarBooking-Frontend)
 
-* Ruby version
+# [Swagger API Doc](https://carbackend.onrender.com/api-docs/index.html)
+![screencapture-localhost-3000-api-docs-index-html-2022-11-29-01_53_41](https://user-images.githubusercontent.com/56429354/204413873-1215219b-6fb2-4d4b-9ece-9d87e658d8d6.png)
 
-* System dependencies
+### Built With
 
-* Configuration
+- Ruby on Rails
+- Development set up
+- Rails API only
+- API Swagger documentation
+- [JWT](https://github.com/jwt/ruby-jwt) & [bcrypt](https://github.com/bcrypt-ruby/bcrypt-ruby) 
 
-* Database creation
+### Setup
 
-* Database initialization
+- Open your terminal and navigate to the folder you would like to save the project.type the following commands to your terminal 
 
-* How to run the test suite
+```
+git clone git@github.com:Gedewon/CarBooking-Backend.git
+```
 
-* Services (job queues, cache servers, search engines, etc.)
+- Move into project directory
 
-* Deployment instructions
+```
+cd CarBooking-Backend
+```
 
-* ...
+### Available Scripts
+
+```
+bundle install && rails db:create db:migrate db:seed 
+```
+
+### Then bootup your database by running the following command 
+
+```
+rails server
+```
+
+
+### Testing
+
+```
+rspec
```

Comment:
- [ ] This command does not work:

![image](https://user-images.githubusercontent.com/57421823/206015412-7d70c975-7ea5-4d53-8906-c573c273a6b7.png)


I recommend changing it for `rake rswag`.  That way, your tests will not fail:

![image](https://user-images.githubusercontent.com/57421823/206015189-b5138c20-c5e0-49eb-bad4-e5a45d5ad1a1.png)

Reply:
@Meltrust Thank you !!